boa/vyper/compiler_utils.py

```diff
@@ -0,0 +1,117 @@
+import textwrap
+
+import vyper.ast as vy_ast
+from vyper.ast.utils import parse_to_ast
+from vyper.ast.signatures.function_signature import FunctionSignature
+from vyper.codegen.function_definitions import generate_ir_for_function
+from vyper.codegen.ir_node import IRnode
+from vyper.exceptions import InvalidType
+from vyper.ir import compile_ir as compile_ir
+import vyper.semantics.validation as validation
+from vyper.semantics.validation.utils import get_exact_type_from_node
+from vyper.utils import abi_method_id
+
+from boa.vyper import _METHOD_ID_VAR
+
+
+def compile_stmt(wrapper_code, contract):
```

Comment:
this seems to be private to this module now. maybe it should be `_compile_stmt`?

Reply:
also it probably needs to be renamed -- it doesn't compile statements anymore, it compiles functions. maybe `_compile_vyper_function` would be better